kustomize/mariadb-cluster/base/mariadb-galera.yaml

```diff
@@ -7,47 +7,75 @@ spec:
   rootPasswordSecretKeyRef:
     name: mariadb
     key: root-password
-
-  database: mariadb
   username: mariadb
-  passwordSecretKeyRef:
-    name: mariadb
-    key: password
-
-  image: mariadb:11.0.3
+  database: mariadb
 
-  port: 3306
+  storage:
+    size: 1Gi
```

Comment:
Storage size should be more than 1G ?

Reply:
I took that value from the new reference  https://github.com/mariadb-operator/mariadb-operator/blob/main/examples/manifests/mariadb_galera_full.yaml#L22 - but we can make it larger.

Reply:
looks like we use 10Gi elsewhere. I'll use that same value here.

Reply:
done.

Reply:
I think maybe they have 1G with the resizeInUseVolumes: true set. Its probably better to have at 10G and not have to resize frequently ?

I was looking at https://github.com/mariadb-operator/mariadb-operator/blob/28bfbe15e1364764c427660bf365dc802fa0f00e/docs/API_REFERENCE.md#storage